linux_os/guide/services/ntp/service_timesyncd_configured/oval/shared.xml

```diff
@@ -0,0 +1,61 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("Ensure that timesyncd is enabled and configured") }}}
+    <criteria comment="Timesyncd servers are configured" operator="AND">
+      <extend_definition comment="service timesyncd enabled"
```

Comment:
In my previous review I mentioned the `service_timesyncd_enabled` and this criterion was simplified. However, I still think it is unnecessary. The service can be configured regardless of being enabled or not. The other rule (service_timesyncd_enabled) should alone ensure it is enabled. Could you consider removing it completely in this configuration rule, please?